language/move-lang/src/expansion/translate.rs

```diff
@@ -877,6 +894,10 @@ fn exp_(context: &mut Context, sp!(loc, pe_): P::Exp) -> E::Exp {
             }
         }
         PE::Annotate(e, ty) => EE::Annotate(exp(context, *e), type_(context, ty)),
+        PE::Spec(spec_block) => {
+            let u = context.bind_exp_spec(spec_block);
```

Comment:
Nit, I would write `EE::Spec(context.bind_exp_spec(spec_block))` so it formats onto one line

Reply:
Done.

Reply:
Done.